ngraph/core/src/op/acos.cpp

```diff
@@ -45,8 +45,12 @@ op::Acos::Acos(const Output<Node>& arg)
 
 shared_ptr<Node> op::Acos::clone_with_new_inputs(const OutputVector& new_args) const
 {
-    check_new_args_count(this, new_args);
-    return make_shared<Acos>(new_args.at(0));
+    NGRAPH_OP_SCOPE(Acos_clone_with_new_inputs)
+    {
+        check_new_args_count(this, new_args);
+        return make_shared<Acos>(new_args.at(0));
+    }
```

Comment:
why do we need to move code inside `{}`?

Reply:
It is needed in order to remove this code if we don't use it.
I changed the semantic of OV_SCOPE macros in the PR https://github.com/openvinotoolkit/openvino/pull/3692

Reply:
@ilya-lavrenov it works like:
```
remove_if (condition) 
{
    code
}
```